src/Lime/Protocol/Command.ts

```diff
@@ -8,6 +8,7 @@ interface Command extends Envelope {
   method: CommandMethod;
   status?: CommandStatus;
   reason?: Reason;
+  timeout?: boolean;
```

Comment:
The timeout property doesn't exists in the Command envelope specification and adding this here can lead to confusion.

Reply:
I inserted this `timeout` property because of new `processCommand` method from `blip-sdk`. Command

```javascript
processCommand(command: Command, timeout = this.commandTimeout): Promise<Command> {
    const commandPromise = Promise.race([
      new Promise((resolve) => {
        this._commandResolves[command.id] = c => {
          if (!c.status) return

          resolve(c)

          delete this._commandResolves[command.id]
        }
      }),
      new Promise((_, reject) => {
        setTimeout(() => {
          if (!this._commandResolves[command.id]) return

          delete this._commandResolves[command.id]
          command.status = 'failure'
          command.timeout = true

          const cmd = JSON.stringify(command)
          reject(new ClientError(cmd))
        }, timeout)
      })
    ])

    this.sendCommand(command)
    return commandPromise
  }
```

Reply:
Should we remove `command.timeout` assignment?